internal/cmd/providers.go

```diff
@@ -245,6 +249,26 @@ $ infra providers add google --url accounts.google.com --client-id 0oa3sz06o6do0
 			}
 
 			cli.Output("Connected provider %q (%s) to infra", args[0], opts.URL)
+
+			if opts.SCIM {
+				key, err := client.CreateAccessKey(&api.CreateAccessKeyRequest{
+					UserID:            provider.ID,
+					Name:              fmt.Sprintf("%s-SCIM", args[0]),
+					TTL:               api.Duration(time.Hour * 87600), // 10 years
+					ExtensionDeadline: api.Duration(time.Hour * 87600), // 10 years
+				})
+				if err != nil {
+					if api.ErrorStatusCode(err) == 403 {
+						logging.Debugf("%s", err.Error())
+						return Error{
+							Message: "Cannot create SCIM key: missing privileges for CreateKey",
+						}
+					}
```

Comment:
I think we don't need this anymore because we return pretty reasonable errors from the api, but I'm ok if we want to keep it.

Reply:
No reason to keep it around if its not useful, removed